migrations/V25__Modify_EUA_Constraint.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE system_intake
+    DROP CONSTRAINT eua_id_check,
+    ADD CONSTRAINT eua_id_check CHECK (public.system_intake.eua_user_id ~ '^[A-Z0-9]{4}$');
```

Comment:
Our previous constraint was not checking that the EUAID was EXACTLY four characters